dashboard/src/actions/overviewActions.js

```diff
@@ -290,17 +305,23 @@ export const editMetadata =
 
     const rIndex = data.findIndex((item) => item.resource_id === rId);
     data[rIndex][metadata] = value;
-    data[rIndex][CONSTANTS.IS_DIRTY] = true;
-    if (value.length > CONSTANTS.DATASET_NAME_LENGTH) {
-      data[rIndex][CONSTANTS.NAME_VALIDATED] = CONSTANTS.ERROR;
-      data[rIndex][
-        CONSTANTS.NAME_ERROR_MSG
-      ] = `Length should be < ${CONSTANTS.DATASET_NAME_LENGTH}`;
-    } else if (value.length === 0) {
-      data[rIndex][CONSTANTS.NAME_VALIDATED] = CONSTANTS.ERROR;
-      data[rIndex][CONSTANTS.NAME_ERROR_MSG] = `Length cannot be 0`;
+
+    if (metadata === "name") {
```

Comment:
`"name"` should be a defined constant, so that both the caller and the callee agree on its value (and to that the interpreter will catch it if either of them misspell it).